app/views/items/index.html.erb

```diff
@@ -6,20 +6,36 @@
 <% end %>
 <section class="grid-container">
   <% @items.each do |item| %>
-    <section class = "grid-item" id= 'item-<%=item.id%>'>
-      <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
-      <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-      <img src= <%= item.image %>>
-      <p> <%= item.description unless @merchant%> </p>
-      <p>Price: <%=number_to_currency(item.price) %> </p>
-      <p>Inventory: <%= item.inventory %> </p>
-      <% if !@merchant %>
-      <% end %>
-      <% if item.active? %>
-        <p>Active</p>
-      <% else %>
-        <p>Inactive</p>
-      <% end %>
-    </section>
+    <% if item.active? %>
+      <section class = "grid-item" id= 'item-<%=item.id%>'>
+        <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
+        <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
+        <%= link_to image_tag(item.image, id: "#{item.name}"), "/items/#{item.id}" %>
+        <p> <%= item.description unless @merchant%> </p>
+        <p>Price: <%=number_to_currency(item.price) %> </p>
+        <p>Inventory: <%= item.inventory %> </p>
+        <% if !@merchant %>
+        <% end %>
+        <% if item.active? %>
+          <p>Active</p>
+        <% else %>
+          <p>Inactive</p>
+        <% end %>
+      </section>
     <% end %>
+  <% end %>
 </section>
+
+<div class=top_5>
+  <%= "Top 5 Most Popular Items:" %>
+  <% @items.top_5.each do |name, quantity| %>
+    <p><%= "#{name} => Total Purchased: #{quantity}\n" %></p>
```

Comment:
Tried to make this print out cool with numbers at the front but gave up. Maybe if we hit this project out of the park with time to spare we can come back to this.